pyrgg/__main__.py

```diff
@@ -87,15 +87,21 @@ def gen_graph(input_dict, file_name):
         elapsed_time_format)
 
 
-def run():
+def run(input_dict=None):
     """
     Run proper converter.
 
+    :param input_dict: input data
+    :type input_dict: dict
     :return: None
     """
-    input_dict = get_input()
+    if input_dict is None:
+        input_dict = get_input()
     file_name = input_dict["file_name"]
     number_of_files = input_dict["number_of_files"]
+    if input_dict["config"] is True:
+        print("Config --> " + save_config(input_dict))
```

Comment:
@sadrasabouri I want to merge this PR, but before that I have a question:
Is it okay to re-save the config file? (when load data from config)
It's a bit weird 😕

Reply:
This way the code architecture remains unified. I believe there is no problem (except facing an error while saving it).

If you are thinking of any better solution please let me know.